components/file_system/src/lib.rs

```diff
@@ -107,6 +126,7 @@ impl std::ops::Sub for IOBytes {
     }
 }
 
+#[repr(u32)]
```

Comment:
Seems u8 is enough. How about use u8 instead?

Reply:
I think we can't really gain from using u8, the vector is wrapped in `CachePadded`, and it's possible that u8 arithmetic is slower, considering most architectures are at least 32bit aligned.